chroma-agent/chroma_agent/device_plugins/linux_components/block_devices.py

```diff
@@ -80,35 +76,35 @@ def get_default(prop, default_value, x):
 
 
 def get_major_minor(x):
-    return "%s:%s" % (x.get('MAJOR'), x.get('MINOR'))
+    return "%s:%s" % (x.get('major'), x.get('minor'))
 
 
 def as_device(x):
-    paths = sort_paths(get_default('PATHS', [], x))
+    paths = sort_paths(get_default('paths', [], x))
```

Comment:
This should be sorted in the scanner now.

Reply:
this is removed later